package.json

```diff
@@ -22,6 +22,7 @@
   "bugs": {
     "url": "https://github.com/learningequality/kolibri/issues"
   },
+  "private": true,
```

Comment:
Just putting this link here for reference: https://docs.npmjs.com/files/package.json#private